testsuite/pytests/test_parameter.py

```diff
@@ -20,108 +20,127 @@
 # along with NEST.  If not, see <http://www.gnu.org/licenses/>.
 
 """
-Simple Parameter tests
-"""
-
-import nest
-import unittest
-
-
-class TestParameter(unittest.TestCase):
-
-    def setUp(self):
-        nest.ResetKernel()
-
-    def test_add(self):
-        low = 55.
-        high = 75.
-        val = 33.
-        e = nest.random.uniform(low, high) + val
-
-        self.assertGreater(e.GetValue(), low + val)
-        self.assertLess(e.GetValue(), high + val)
-
-    def test_radd(self):
-        low = 55.
-        high = 75.
-        val = 33.
-        e = val + nest.random.uniform(low, high)
-
-        self.assertGreater(e.GetValue(), low + val)
-        self.assertLess(e.GetValue(), high + val)
-
-    def test_sub(self):
-        low = 55.
-        high = 75.
-        val = 33.
-        e = nest.random.uniform(low, high) - val
-
-        self.assertGreater(e.GetValue(), low - val)
-        self.assertLess(e.GetValue(), high - val)
-
-    def test_rsub(self):
-        low = 55.
-        high = 75.
-        val = 33.
-        e = val - nest.random.uniform(low, high)
-
-        self.assertGreater(e.GetValue(), val - high)
-        self.assertLess(e.GetValue(), val - low)
-
-    def test_neg(self):
-        low = 55.
-        high = 75.
-        e = -nest.random.uniform(low, high)
+Test that Parameters support arithmetic operations correctly.
 
-        self.assertGreater(e.GetValue(), -high)
-        self.assertLess(e.GetValue(), -low)
-
-    def test_rsub_all_neg(self):
-        low = -55.
-        high = -75.
-        val = -33.
-        e = val - nest.random.uniform(high, low)
-
-        self.assertGreater(e.GetValue(), val - low)
-        self.assertLess(e.GetValue(), val - high)
-
-    def test_mul(self):
-        low = 55.
-        high = 75.
-        val = 3.
-        e = nest.random.uniform(low, high) * val
-
-        self.assertGreater(e.GetValue(), low * val)
-        self.assertLess(e.GetValue(), high * val)
-
-    def test_rmul(self):
-        low = 55.
-        high = 75.
-        val = 3.
-        e = val * nest.random.uniform(low, high)
-
-        self.assertGreater(e.GetValue(), val * low)
-        self.assertLess(e.GetValue(), val * high)
-
-    def test_div(self):
-        low = 55.
-        high = 75.
-        val = 3.
-        e = nest.random.uniform(low, high) / val
-
-        self.assertGreater(e.GetValue(), low / val)
-        self.assertLess(e.GetValue(), high / val)
-
-
-def suite():
-    suite = unittest.makeSuite(TestParameter, 'test')
-    return suite
-
-
-def run():
-    runner = unittest.TextTestRunner(verbosity=2)
-    runner.run(suite())
+This set of tests confirms that arithmetic operations on NEST Parameter objects yield
+correct results by comparing with operations on the undelying numeric values.
+It also confirms that operations on Parameter objects and plain numbers work.
 
+@note These tests only confirm that operators on parameters work in principle. Therefore,
```

Comment:
Isn't `@note` more of a Doxygen command? I don't mind the usage, just wondering if there is a more appropriate way to highlight notes. For instance, my editor highlights `NOTE` (and also `TODO`) in Python docstrings but not `@note`.

Reply:
I agree. I think you can use proper Restructured Text markup in docstrings, as those also get rendered nicely when we include the tests in the documentation (which we probably should at one point).

Reply:
Fixed.